hera_mc/geo_handling.py

```diff
@@ -115,17 +116,19 @@ def cofa(self, show_cofa=False):
         ------------
         show_cofa:  boolean to either show cofa or not
         """
-
         self.get_station_types(add_stations=True)
         current_cofa = self.station_types['COFA']['Stations']
-        located = self.get_location(current_cofa, 'now', show_cofa, 'm')
+        located = self.get_location(current_cofa, 'now', self.station_types)
         if len(located) == 0:
             located_cofa = None
         elif len(located) > 1:
-            print("Warning:  {} has returned multiple cofa values.  Returning None.")
+            s = "{} has multiple cofa values.".format(str(current_cofa))
+            warnings.warn(s)
```

Comment:
I've gone back and forth on this, but I agree with you.  It can give a warning.warn but return the first one.
I'm about to push up a small PR for making the parts stuff case-insenstive and I'll include this change there.  Thanks.